pos_pricelist/static/src/js/models.js

```diff
@@ -162,17 +162,26 @@ function pos_pricelist_models(instance, module) {
          * @param quantity
          */
         set_quantity: function (quantity) {
-            OrderlineParent.prototype.set_quantity.apply(this, arguments);
             var partner = this.order ? this.order.get_client() : null;
             var product = this.product;
             var db = this.pos.db;
+            var old_price = 0;
+            if (this.get_quantity()) {
+                old_price = this.pos.pricelist_engine.compute_price_all(
```

Comment:
No, because then you get the manually modified price instead of the pricelist price.